Scripts/public-api-diff/Sources/Helpers/PackageFileHelper.swift

```diff
@@ -0,0 +1,183 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+struct PackageFileHelper {
```

Comment:
It is not clear which package unless you scan the code and see `Package,swift`
However, I am not sure if `SPMConfigHelper` is a any better?

Reply:
Mhh I added some documentation - maybe that helps?

Reply:
Yes!